components/consumers/pdf/default.html

```diff
@@ -74,110 +113,152 @@
-                    <p>10</p>
-                </div>
-                <div>
-                    <h3>Total High Severity Findings</h3>
-                    <p>10</p>
-                </div>
-            </div>
 
-            <!-- Placeholder Graph -->
-            <div class="placeholder-graph"></div>
+<div class="report">
+    <div class="summary">
+        <p class="introduction">This report summarizes the results of running Smithy.</p>
 
-            <p>The vulnerability scans have identified potential issues that need attention. It is recommended to review
-                and address the findings promptly to enhance the security of our systems.</p>
+        <!-- Placeholder Metrics -->
+        <div class="placeholder-metrics">
+            <div class="total"><h3><span class="total">30</span> Total Findings</h3></div>
+            <div class="high-severity"><h3><span class="total">10</span> High Severity Findings</h3></div>
         </div>
+        <p>The vulnerability scans have identified potential issues that need attention. <br/>It is recommended to
+            review
+            and address the findings promptly to enhance the security of our systems.</p>
 
-        <!-- Scan -->
-        {{range .}}
-        <div class="scan">
-            <div class="scan-title">{{.OriginalResults.ScanInfo.ScanUuid}} - {{.OriginalResults.ToolName}}</div>
-            <div class="scan-details">
-                <div><strong>Start Time:</strong> {{.OriginalResults.ScanInfo.ScanStartTime}}</div>
-            </div>
+        <table class="summary-table">
+            <thead>
+            <tr>
+                <th>Name</th>
+                <th>Severity</th>
+            </tr>
+            </thead>
+            <tbody>
+            {{range .}}
             {{range .Issues}}
-            <div class="finding">
-                <div class="finding-title">{{.RawIssue.Title}}</div>
-                <div class="finding-details">
-                    <div>Target: {{.RawIssue.Target}} </div>
-                    <div>Type: {{.RawIssue.Type}} </div>
-                    <div>CVSS: {{.RawIssue.Cvss}} </div>
-                    <div>CVE: {{.RawIssue.Cve}} </div>
-                    <div>Confidence: {{.RawIssue.Confidence}} </div>
-                    <div>Severity: {{.RawIssue.Severity}} </div>
-                    <div>Description: {{.RawIssue.Description}} </div>
-                    <div>First Seen: {{.FirstSeen}} </div>
-                    <div>Seen Before Times: {{.Count}} </div>
-                    <div>False Positive?:{{.FalsePositive}} </div>
-                    <div>Last Updated: {{.UpdatedAt}} </div>
-                    {{ range $key,$element := .Annotations }}
-                    <p><b>{{$key}}</b>:{{$element}}</p>
-                    {{end}}
-                    <div>SBOM<pre>{{.RawIssue.CycloneDXSBOM}}</pre></div>
-                </div>
-            </div>
+            <tr>
+                <td>{{.RawIssue.Title}}</td>
```

Comment:
Can I suggest we use this syntax JIC some vals are empty?

```
{{ if and .RawIssue.Title (ne .RawIssue.Title "") }}{{ .RawIssue.Title }}{{ else }}No Title{{ end }}
```

This is just an example but it would be nice to come up with defaults if we have empty strings